sceptre/cli/launch.py

```diff
@@ -33,6 +42,20 @@ def launch_command(ctx, path, yes):
     )
 
     plan = SceptrePlan(context)
+    plan.resolve(plan.launch.__name__)
+
+    stacks_to_exclude = []
+    for stacks in plan.launch_order:
+        for stack in stacks:
+            if stack.launch_action == LaunchAction.exclude:
+                stacks_to_exclude.append(stack)
```

Comment:
I'm wondering what happens if an excluded stack is a dependency of another stack?  if sceptre deletes it without regard for the dependency wouldn't that mess up downstream stacks?

Reply:
Great question. I think I'll need to research that a little bit. I think the behavior I'd desire would be that if another stack depends on an excluded stack, it would blow up and disallow that. Like, if you're going to exclude a dependency, you need to also exclude everything that depends on it and so on up the hierarchy. I don't think I've accounted for that yet, actually.